### ISSUE TITLE
Lang config on file commitlinterrc.json not working

### ISSUE THREAD

**Issue by leocds13** [legend80s/commit-msg-linter#27]
I configure the config file to pt-BR but the message continue in english:

my commitlinterrc.json:
```
{
  "lang": "pt-BR",
  "min-len": 8,
  "max-len": 100,
  "showInvalidHeader": false
}
```

The output:
![image](https://user-images.githubusercontent.com/28991195/217828285-f635b271-a2f7-4ae1-aba1-930d2bf82b88.png)

I think a found the problem.
On the getLanguage function it converts to Lower:
```
/**
 * @returns {string}
 */
function getLanguage(configLang) {
  const lang =
    configLang ||
    process.env.COMMIT_MSG_LINTER_LANG ||
    Intl.DateTimeFormat().resolvedOptions().locale;

  return lang && lang.toLowerCase();
}
```

So "pt-BR" ends up as "pt-br"
and it was not found in i18n because i18n["pt-BR"] does not exist
![image](https://user-images.githubusercontent.com/28991195/217829170-d58ef123-fb5b-4ed5-9bcc-8e2fa846ba0d.png)

**Comment by leocds13** [comment]
I think the same can happen with other languages

**Comment by legend80s** [comment]
Thanks very much! Fixed at latest git-commit-msg-linter@4.5.1
